src/context.rs

```diff
@@ -62,23 +62,7 @@ use crate::{chatlist_events, stats};
 /// # });
 /// ```
 ///
-/// To use an encrypted database provide a password.  If the database does not yet exist it
-/// will be created:
-///
-/// ```
-/// # let rt = tokio::runtime::Runtime::new().unwrap();
-/// # rt.block_on(async move {
-/// use deltachat::context::ContextBuilder;
-///
-/// let dir = tempfile::tempdir().unwrap();
-/// let context = ContextBuilder::new(dir.path().join("db"))
-///      .with_password("secret".into())
-///      .open()
-///      .await
-///      .unwrap();
-/// drop(context);
-/// # });
-/// ```
+/// NB: Database encryption is deprecated.
```

Comment:
This comment here seems unnecessary, just no example for database encryption is fine.